cypress/TestCases/FireboltCertification/UserGrants.feature

```diff
@@ -0,0 +1,788 @@
+        Then 'Firebolt' platform responds with 'washington for localization locality'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for locality'
+        Then 'Firebolt' platform responds with 'true for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+        #Case-2
+        When User 'starts' recording lifecycle history for '3rd party app'
+        And '3rd party app' invokes the 'Firebolt' API to 'get locality'
+        Then 'Firebolt' platform responds with 'washington for localization locality'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'EMPTY_HISTORY'
+
+    # If lifespan value forever only once ask for grant until we clear the given grant
+    # Case-1: Denying the grant and validating for error
+    # Case-2: Checking the lifecycle history to check ui is not coming and calling postal code and validating for error because already denied grant in Case-1.
+    @Usergrants @coreSDK @sdk @transport @requiresPlatformImplementation
+    Scenario: UserGrants.Capabilities R*2.2.2.2 - Negative Scenario: Validate Capabilities with denied (pinChallenge) and lifespan forever
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        #Case-1
+        And Framework registers 'pinChallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        When '3rd party app' invokes the 'Firebolt' API to 'expect error localization locality'
+        Then 'Firebolt' platform responds with 'invalid request for localization locality'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for locality'
+        Then 'Firebolt' platform responds with 'false for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+        #Case-2
+        When User 'starts' recording lifecycle history for '3rd party app'
+        And '3rd party app' invokes the 'Firebolt' API to 'expect error localization locality'
+        Then 'Firebolt' platform responds with 'invalid request for localization locality'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'EMPTY_HISTORY'
+
+    # If lifespan value appActive it ask grant once when app is active and it ask again when app is closed and active again.
+    # Case-1: Allowing grant when app is active and validating
+    # Case-2: Closing and launching the app, allowing grant and validating ui is coming again
```

Comment:
which glue are we talking about in below TC which is not implemented ?

Reply:
removed the comment